app/dev-app-update.yml

```diff
@@ -1,3 +1,3 @@
-owner: LiskHQ
+owner: reyraa
```

Comment:
should this be LiskHQ?

Reply:
For sure. But shouldn't we resolve it in your PR?

Reply:
yes, i think we should just keep only BTC changes in this PR